core/src/main/java/org/infinispan/functional/impl/MetaParamsInternalMetadata.java

```diff
@@ -113,60 +123,32 @@ NumericVersion getNumericVersion() {
       return version instanceof NumericVersion ? (NumericVersion) version : null;
    }
 
-   void setNumericVersion(NumericVersion version) {
-      params.add(new MetaEntryVersion(version));
-   }
-
    @ProtoField(number = 2)
    SimpleClusteredVersion getClusteredVersion() {
       EntryVersion version = version();
       return version instanceof SimpleClusteredVersion ? (SimpleClusteredVersion) version : null;
    }
 
-   void setClusteredVersion(SimpleClusteredVersion version) {
-      params.add(new MetaEntryVersion(version));
-   }
-
-   @ProtoField(number = 3, defaultValue = "-1")
+   @ProtoField(number = 3, defaultValue = "0")
```

Comment:
Ooops, my previous comment about 0 default here was very wrong. If we make it 0, then a missing 'created' field in data stream will become 0 and the constructor will not see it as being ```< -1``` so it will create a MetaCreated param out of the blue, which is obviously unintended.

Reply:
No need to update your PR @ryanemerson . I'll review all defaults in this PR and add more comments and after we figure them out I'll fix and merge.